test/vstest.console.UnitTests/TestPlatformHelpers/TestRequestManagerTests.cs

```diff
@@ -2571,4 +2573,50 @@ private void DiscoverTestsIfThrowsExceptionShouldThrowOut(Exception exception)
 
         _testRequestManager.DiscoverTests(payload, mockDiscoveryEventsRegistrar.Object, _protocolConfig);
     }
+
```

Comment:
Please add a test for the `SettingsException`

Reply:
Added. UsingInvalidValueForDefaultPlatformSettingThrowsSettingsException